Dockerfile

```diff
@@ -16,11 +16,12 @@ RUN chmod a=r /usr/local/share/amazon-certs.pem
 RUN pip install -U pip
 RUN pip install "poetry==1.6.1"
 
-WORKDIR /dmap_importer
-
-# copy pyproject, readme, and source code into the container
+# copy poetry and pyproject files and install dependencies
+WORKDIR /dmap_import/
 COPY pyproject.toml pyproject.toml
-COPY src src
+RUN poetry install --no-dev --no-interaction --no-ansi -v
 
-# install python dependencies and build project
+# Copy src directory to run against and build app
+COPY src src
+COPY alembic.ini alembic.ini
 RUN poetry install --no-dev --no-interaction --no-ansi -v
```

Comment:
This order of commands allows for changes to library files without having to re-install all dependencies (if they didn't change)